src/crypto/password/bcrypt.clj

```diff
@@ -1,20 +1,20 @@
 (ns crypto.password.bcrypt
   "Functions for encrypting passwords using the widely-used bcrypt algorithm.
 
-  See: http://bcrypt.sourceforge.net/"
-  (:import org.mindrot.jbcrypt.BCrypt))
+  See: https://github.com/patrickfav/bcrypt"
+  (:import [at.favre.lib.crypto.bcrypt BCrypt]))
 
 (defn encrypt
   "Encrypt a password string using the BCrypt algorithm. The optional work
   factor is the log2 of the number of hashing rounds to apply. The default
-  work factor is 11."
+  work factor is 12."
```

Comment:
What's the reason behind this change?

Reply:
I tried considering the recommendations by OWASP https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Password_Storage_Cheat_Sheet.md#work-factor having in mind this benchmarks: https://wildlyinaccurate.com/bcrypt-choosing-a-work-factor/

12 Rounds is the default by BSDs PassLib https://passlib.readthedocs.io/en/stable/lib/passlib.hash.bcrypt.html

In case of doubt I vote for a conservative default

Reply:
As mentioned in the README, the work factor for crypto-password was chosen by benchmarking on AWS machines. I don't mind changing it, but it should be a separate PR that updates all of the work factors, based on a new set of benchmarks.

Reply:
ok